science/madness/Portfile

```diff
@@ -20,9 +20,9 @@ long_description    MADNESS provides a high-level environment for the solution \
                     of integral and differential equations in many dimensions \
                     using adaptive, fast methods with guaranteed precision \
                     based on multi-resolution analysis and novel separated representations.
-checksums           rmd160  d9241d1bc207e12ea857938cf1ac111ce2acba83 \
-                    sha256  bbef0cacd92c5bbf546212914d11c269a2e6224078a2f67e43e2a58b5e2803df \
-                    size    36799675
+checksums           rmd160  1689838da6e097d2bd4df1d1e8f7f10382b0868b \
+                    sha256  3b7bea52d12efb74311a3744a23431a836378d1f0762ef79337418530307c5b0 \
+                    size    36804651
 github.tarball_from archive
 
 set py_ver          3.11
```

Comment:
Is 3.12 possible here?

Reply:
Undesirable, but I am kinda tired of arguing over pythons, and for non-essential ports it is easier to just give up and change it to whatever LOL

Reply:
Ah OK, I always forget that there are issues with Python versions in the PPC/older macOS world.

Reply:
No big deal here, let it be on 3.12.